remote-playback/remote-video-control-pausing-manual.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+  <title>Test if video is playing on remote device.</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/common/media.js"></script>
+  <style>
```

Comment:
This is a super minor thing, but consider putting this in an external stylesheet so it can be used by all the tests.